sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -717,7 +717,7 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       where: {
         id: institutionId,
       },
-      cache: true,
+      // TODO:Implement the cache again after updating cache eviction strategy.
```

Comment:
Did we add acceptance criteria to have the cache added back in another ticket?

Reply:
Yes it was added during the grooming.

![image](https://github.com/bcgov/SIMS/assets/54600590/160a4770-29cb-44e3-800d-ab197ae46fde)

Reply:
My point being, if it is there we do not need a TODO in the code 😉